openshift/factstore.app.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: template.openshift.io/v1
```

Comment:
This file should not be part of this PR as there is a separate PR for it. Or you should close that PR in favour of this one

Reply:
I've removed this from this PR.